### PR TITLE
Feature/par 84 uiux create dropdown component

### PR DESCRIPTION
**Pull request by Sathwik-Amburi**
Screenshot from 2023-11-20 17-45-18

Implemented a Simple Dropdown. Everything has been abstracted for simplicity. It only requires 'onClick' and 'menu items' as props, along with the dropdown text.

### CODE REVIEW

**Comment by Sathwik-Amburi** [comment]
If we decide to use the Select Component instead then we can use the component right outside the box (https://ant.design/components/select)

File: src/app/simulations/page.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+'use client';
```
Comment (GLDuval) [inline src/app/simulations/page.tsx:1]: Can remove this now too

**Review by GLDuval — COMMENTED** [review]
Looks good!

**Review by guid3d — APPROVED** [review]

File: src/components/generic/Dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import type { MenuProps } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+
+interface DropdownProps {
+  menu: MenuProps;
+  text: string;
+}
+
+// eslint-disable-next-line require-jsdoc
+export const DropdownMenu = (props: DropdownProps) => {
```
Comment (guid3d) [inline src/components/generic/Dropdown.tsx:12]:
```suggestion
const DropdownMenu = (props: DropdownProps) => {
```

File: src/components/generic/Dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <a onClick={e => e.preventDefault()}>
+          <Space>
+            {props.text}
+            <DownOutlined />
+          </Space>
+        </a>
+      </Dropdown>
+    </>
+  );
+};
```
Comment (guid3d) [inline src/components/generic/Dropdown.tsx:25]:
```suggestion
};

export default DropdownMenu
```

File: src/components/generic/InputField.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  disabled?: boolean;
+  showCount?: boolean;
+  rows?: number;
+  iconRender?: () => React.ReactNode;
+}
+
+const { TextArea } = Input;
+const { Password } = Input;
+// eslint-disable-next-line require-jsdoc
+export function InputField(props: InputFieldProps) {
```
Comment (guid3d) [inline src/components/generic/InputField.tsx:23]:
```suggestion
const InputField = (props: InputFieldProps) => {
```

File: src/components/generic/InputField.tsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+          onChange={props.onChange}
+          onPressEnter={props.onPressEnter}
+          maxLength={props.maxLength}
+          disabled={props.disabled}
+          showCount={props.showCount}
+        />
+      </>
+    );
+  }
+}
```
Comment (guid3d) [inline src/components/generic/InputField.tsx:77]:
```suggestion
}

export default InputField
```

**Review by guid3d — DISMISSED** [review]

**Review by guid3d — APPROVED** [review]

**Pull request merged**